pkg/kubelet/dockershim/docker_service.go

```diff
@@ -197,6 +197,15 @@ func NewDockerService(client dockertools.DockerInterface, seccompProfileRoot str
 	} else {
 		cgroupDriver = dockerInfo.CgroupDriver
 	}
+
+	if kubeEnableCRI {
```

Comment:
Why is this relevant only with the CRI? Even without it we blindly created symlinks assuming the json files would be available.